modules/storage/storage-account/test/main.test.bicep

```diff
@@ -13,10 +19,50 @@ param location string = resourceGroup().location
 //   }
 // }
 
-//Test 0. 
+module prerequisites 'prereq.test.bicep' = {
+  name: 'test-prereqs'
+  params: {
+    name: serviceShort
+    location: location
+    prefix: uniqueName
+  }
+}
+
+//Test 0.
 module test0 '../main.bicep' = {
```

Comment:
create a new test case for this. We want to keep test0 as is requiring the minimum amount of parameters.